crypto/modes/siv128.c

```diff
@@ -223,10 +235,11 @@ int CRYPTO_siv128_aad(SIV128_CONTEXT *ctx, const unsigned char *aad,
 
     siv128_dbl(&ctx->d);
 
-    mac_ctx = EVP_MAC_CTX_dup(ctx->mac_ctx_init);
-    if (mac_ctx == NULL
+    ;
```

Comment:
?

Reply:
I think I had some debugging code there, and didn't clean it well enough.  Done